lib/v2/science/blogs.js

```diff
@@ -0,0 +1,55 @@
+const cheerio = require('cheerio');
+const { parseDate } = require('@/utils/parse-date');
+const { baseUrl } = require('./utils');
+
+module.exports = async (ctx) => {
+    const { name = 'pipeline' } = ctx.params;
+    const link = `${baseUrl}/blogs/${name}/feed`;
+
+    const response = await ctx.cache.tryGet(link, async () => {
+        const browser = await require('@/utils/puppeteer')();
+        const page = await browser.newPage();
+        await page.setRequestInterception(true);
+
+        page.on('request', (request) => {
+            request.resourceType() === 'document' ? request.continue() : request.abort();
+        });
+
+        await page.goto(link, {
+            waitUntil: 'domcontentloaded',
+        });
+
+        const response = await page.content();
+
+        page.close();
+        browser.close();
+        return response;
+    });
```

Comment:
```js
const config = require('@/config').value;
```
```suggestion
    }, config.cache.routeExpire, false);
```
Otherwise, the entries won't receive updates.